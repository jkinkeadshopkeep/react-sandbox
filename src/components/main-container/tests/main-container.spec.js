import React from 'react';
import { render } from '@testing-library/react';
import MainContainer from "../index";

describe('Main container Component', () => {
  it('should render with content inside container', () => {
    const { container } = render(<MainContainer><p>Hello am rendered</p></MainContainer>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
