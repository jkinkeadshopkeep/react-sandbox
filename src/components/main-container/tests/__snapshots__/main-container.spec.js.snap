// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main container Component should render with content inside container 1`] = `
<div
  class="main-container"
>
  <p>
    Hello am rendered
  </p>
</div>
`;
